src/js/bootstrap-slider.js

```diff
@@ -1261,6 +1269,16 @@
 
 				return true;
 			},
+			_touchstart: function(ev) {
+				if (ev.changedTouches === undefined) {
+					this._trigger('touchStart');
```

Comment:
could you have this trigger `slideStart` instead of `touchStart` as that is the event we have defined in our README?